generators/typescript/express/generator/src/ExpressGenerator.ts

```diff
@@ -564,7 +564,9 @@ export class ExpressGenerator {
             includeSerdeLayer: this.config.includeSerdeLayer,
             retainOriginalCasing: this.config.retainOriginalCasing,
             useBigInt: this.config.useBigInt,
-            enableInlineTypes: false
+            enableInlineTypes: false,
+            allowExtraFields: this.config.allowExtraFields,
+            omitUndefined: false
```

Comment:
Is `this.config.omitUndefined` not available here?

Reply:
No, it's not part of the express config. We can add it in the future if we ever need it.
I defaulted to the default value that is in the SDK config.